src/app/organizer/Columns.tsx

```diff
@@ -535,6 +536,21 @@ export function getColumns(
         header: t('Organizer.Columns.MasterworkStat'),
         value: (item) => getMasterworkStatNames(item.masterworkInfo),
       }),
+    destinyVersion === 2 &&
+      isWeapon &&
+      c({
+        id: 'deepsightLevel',
+        header: t('Organizer.Columns.DeepsightLevel'),
+        value: (item) => item.craftedInfo?.level,
+        defaultSort: SortDirection.DESC,
+      }),
+    destinyVersion === 2 &&
+      isWeapon &&
+      c({
+        id: 'deepsightHarmonizable',
+        header: t('Organizer.Columns.DeepsightHarmonizable'),
+        value: (item) => (isHarmonizable(item) ? t('Organizer.Yes') : t('Organizer.No')),
```

Comment:
The value should be the boolean (true or false) and then the cell should be the actual content.